doc/users/next_whats_new/2017-12-27_legend_title_size_rc.rst

```diff
@@ -0,0 +1,6 @@
+Legend Title Size rc parameter
+------------------------------
+
+A new rc parameter has been added as an option in your matplotlibrc allowing you to control the font size of the legend title.
+
+`legend.titlesize = 'large'`
```

Comment:
This is a little ambiguous to a new user as to where they would enter this command.  I'd be more explicit with a bit of code:

```
.. code-block:: python
    import matplotlib.rcParams as rcParams
    
    rcParams['legend.titlesize'] = 'large'
```